rubygem-smart_proxy_dns_route53/rubygem-smart_proxy_dns_route53.spec

```diff
@@ -0,0 +1,77 @@
+# Generated from smart_proxy_dns_route53-2.0.0.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
Looks like this is using the wrong template, it should use the smart proxy plugin template which installs bundler.d files etc.